README.md

```diff
@@ -271,16 +270,16 @@ Compiling On Linux
    ```bash
    mkdir build & cd build
    cmake ../engine
-   make -j $(nproc) # (where $(nproc) returns the number of available CPU threads/cores on the system)
+   cmake --build ./build -j $(getconf _NPROCESSORS_ONLN) # (where the getconf clause returns the number of available CPU threads/cores on the system)
    mkdir ../bin && cp vegastrike-engine ../bin/ && cp setup/vegasettings ../bin/ && cd ..
    ```
 
    __TIPS__:
 
-   To enable verbose output for debugging purposes (will show compilation commands), pass the `VERBOSE=1` argument:
+   To enable verbose output for debugging purposes (will show compilation commands), pass the `--version` argument, where supported:
```

Comment:
`--version` here should be `--verbose`, right?

Reply:
@P-D-E Oops, you're right. Let me fix that

Reply:
Fixed